# HG changeset patch
# User Ivan Topolcic <topolcic.ivan@gmail.com>
# Date 1551920098 18000
#      Wed Mar 06 19:54:58 2019 -0500
# Node ID 599aa6e009c66e78e7ce4239a8f3b9487c24a7d7
# Parent  4bf562c82eca807581df594f7ed2a238c03dae9e
Changed deprecated waitForCondition calls with TestUtils.waitForCondition

diff --git a/browser/components/customizableui/test/browser_913972_currentset_overflow.js b/browser/components/customizableui/test/browser_913972_currentset_overflow.js
--- a/browser/components/customizableui/test/browser_913972_currentset_overflow.js
+++ b/browser/components/customizableui/test/browser_913972_currentset_overflow.js
@@ -18,12 +18,12 @@ add_task(async function() {
   let navbarTarget = CustomizableUI.getCustomizationTarget(navbar);
   let oldChildCount = navbarTarget.childElementCount;
   window.resizeTo(kForceOverflowWidthPx, window.outerHeight);
-  await waitForCondition(() => navbar.hasAttribute("overflowing"));
+  await TestUtils.waitForCondition(() => navbar.hasAttribute("overflowing"));
   ok(navbar.hasAttribute("overflowing"), "Should have an overflowing toolbar.");
   ok(CustomizableUI.inDefaultState, "Should still be in default state when overflowing.");
   ok(navbarTarget.childElementCount < oldChildCount, "Should have fewer children.");
   window.resizeTo(originalWindowWidth, window.outerHeight);
-  await waitForCondition(() => !navbar.hasAttribute("overflowing"));
+  await TestUtils.waitForCondition(() => !navbar.hasAttribute("overflowing"));
   ok(!navbar.hasAttribute("overflowing"), "Should no longer have an overflowing toolbar.");
   ok(CustomizableUI.inDefaultState, "Should still be in default state now we're no longer overflowing.");